### PR TITLE
A22-AB2-30

### PR DESCRIPTION
**Pull request by ndwilliamson123**
Finished Selected Warehouse component with responsive styling. Minor changes to React-Router and Footer.

### CODE REVIEW

File: src/components/SelectedItem/SelectedItem.js [hunk 1]
```diff
@@ -1,4 +1,71 @@
-export default function SelectedItem() {
-  console.log("selected item");
-  return <h1>Display Selected Item Info</h1>;
+import "./SelectedItem.scss";
+import { Component } from "react";
+import { DetailsSubHeader } from "../index";
+
+//Temporary import of inventory data until backend is setup
```
Comment (BrandonWong22) [inline src/components/SelectedItem/SelectedItem.js:5]: can this be merged in atm if the backend is not setup?

File: src/components/SelectedWarehouseInventoryListColumns/SelectedWarehouseInventoryListColumns.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import "./SelectedWarehouseInventoryListColumns.scss";
+import sortSymbol from "../../assets/icons/sort-24px.svg";
+
+export default function SelectedWarehouseInventoryListColumns() {
+    return (
+        <div className="selected-warehouse-inventory-list__columns">
+            <div>
+                <h3>INVENTORY ITEM</h3>
+                <img src={sortSymbol} alt="" />
```
Comment (BrandonWong22) [inline src/components/SelectedWarehouseInventoryListColumns/SelectedWarehouseInventoryListColumns.js:9]: alt tags should not be left blank

File: src/components/WarehousesList/WarehousesList.js [hunk 3]
```diff
@@ -2,53 +2,56 @@ import "./WarehousesList.scss";
 import { Component } from "react";
```
Comment (BrandonWong22) [inline src/components/WarehousesList/WarehousesList.js:2]: did anything change in this file? looks like the only diff is due to spacing

File: src/pages/Warehouses/Warehouses.js [hunk 4]
```diff
@@ -1,31 +1,31 @@
 import { Switch, Route } from "react-router-dom";
```
Comment (BrandonWong22) [inline src/pages/Warehouses/Warehouses.js:1]: same here, what has changed here besides spacing?

File: src/components/DetailsSubHeader/DetailsSubHeader.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import "./DetailsSubHeader.scss";
+import { Link } from "react-router-dom";
+import backSymbol from "../../assets/icons/arrow_back-24px.svg";
+import editSymbol from "../../assets/icons/edit-24px-white.svg";
+
+export default function DetailsSubHeader({ title, buttonText, id, warehouseOrInventory }) {
+    return (
+        <div className="details-header__container">
+            <div>
+                <Link to={`/${warehouseOrInventory}`}>
```
Comment (SomeViet) [inline src/components/DetailsSubHeader/DetailsSubHeader.js:10]: Minor comment: Should this go to warehouse or inventory. Pick one for now, and if we have time, we can figure out how to implement "back history"

**Review by SomeViet — APPROVED** [review]
Approved. Ticket #30 completed. Please keep Ticket #17 open until API is connected.

File: src/components/SelectedItem/SelectedItem.js (hunk 1 above)
Comment (SomeViet) [reply src/components/SelectedItem/SelectedItem.js:5]: I've approved the merge, we will merge to get the front-end in at least, and we will keep the ticket open until API is hooked up.

**Pull request merged**